ansible/rancher/README.md

```diff
@@ -0,0 +1,52 @@
+# Running the Rancher Playbook
```

Comment:
It might be helpful to include an example in this at some point (with dummy values instead of any sensitive info, if there is any required). This is great for right now though!